lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -83,6 +83,21 @@ def test_permissions(self):
         resp = self.client.get(self.get_url(self.student.username))
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_not_active_user_access(self):
+        """
+        Not active users are allowed to access
```

Comment:
Verify inactive users—those who have not verified their email addresses—are allowed to access the endpoint.